src/command.ts

```diff
@@ -319,6 +319,13 @@ export class Command<C extends Context = Context> implements MiddlewareObj<C> {
             .use(...middlewareArray);
           break;
         case "chat":
+          if (scope.chat_id) {
+            this._composer
+              .filter(Command.hasCommand(this.names, optionsObject))
+              .filter((ctx) => ctx.chat?.id === scope.chat_id)
```

Comment:
Is there a reason why the plugin doesn't use
```suggestion
              .filter((ctx) => ctx.chatId === scope.chat_id)
```
everywhere? Is it because the extra cases were ruled out manually? Is it because of compatibility with an old version of grammY? Is it just an oversight?

Reply:
Just an oversight